signal/signals_library.go

```diff
@@ -0,0 +1,101 @@
+// +build library,!darwin
```

Comment:
`!android`?

Reply:
So `signals_library.go` is for both Desktop and Android builds.

It's interesting how only three dimensions of build options can create so much confusion :)